### ISSUE TITLE
Installation issues: incompatible TargetFramework

### ISSUE THREAD

**Issue by stefanloerwald** [excubo-ag/WebCompiler#13]
A common issue people experience is that supposedly the package is incompatible with their projects. This happens because installing the tool via the nuget package manager for a project is possible (because nuget allows it), but not the correct way to install. See [Getting started](https://github.com/excubo-ag/WebCompiler#getting-started) for the correct way of installing this tool.

**Comment by elken** [comment]
Possible to create a LibWebcompiler and have the exe inherit it?

**Comment by stefanloerwald** [comment]
It is possible to create a nuget package compatible with projects for this, but it would need to be a different nuget package anyway.

I guess the easiest way would be to create a build task that takes care of installing webcompiler and running it, but it would need to come with a few assumptions (e.g. run recursively on wwwroot, but not everbody wants that). To have webcompiler be highly configurable, we would need to have configuration options easily accessible. That's not as trivial with a build task. A config file would be possible, but that would need to be spec'ed and maintained.

So long story short: I'm much more in favor of people reading the Readme or simply come here to be pointed to the Readme.
